demos/simple/output.html

```diff
@@ -7,12 +7,12 @@
 <body>
     <div id="live-editor-output"></div>
 
-    <script src="../../build/js/live-editor.core_deps.js"></script>
+    <!-- TODO(kevinb7) use JSON dependency loading system for core_deps.js and shared.js -->
+    <script src="../../build/js/live-editor.core_deps.js?cachebust=1"></script>
```

Comment:
I would've thought so too.  Chrome was being quite silly.  It may have also been the local server that I was using.  It was really weird... Chrome would load the correct version in index.html and then load an old cached one in output.html.